### DISCUSSION TITLE
Query string with multiple words seem to trigger separate searches?

### DISCUSSION THREAD

**Discussion by peirix** [maize-tech/laravel-searchable#15] [Q&A]
I've implemented this package in a product search for a customer, and we find that since some products have variation names with just a couple of letters this seems to mess up the search a bit.

Example product name: «Crampler Border AG», so if the client wants to search for that it will return any product that has «ag» somewhere in the title, SKU, description or type. Which is a lot. Is there a way to have the query only return products that have hits in all the words of the query?

Realised as I'm typing now that this was a bit difficult to explain. I hope this made sense?

**Comment by riccardodallavia** [comment]
Hi @peirix

I got your question and it totally makes sense!
We currently do not support the ability to only return the exact query searched, but it could easily be added to the package. Want to try to create a PR for that?
